modin/pandas/test/test_series.py

```diff
@@ -4258,11 +4259,28 @@ def test_encode(data, encoding_type):
 
 
 @pytest.mark.parametrize("data", test_string_data_values, ids=test_string_data_keys)
-def test_add_string_to_series(data):
+def test_non_commutative_add_string_to_series(data):
+    # This test checks that add and radd do different things when addition is
+    # not commutative, e.g. for adding a string to a string. For context see
+    # https://github.com/modin-project/modin/issues/4908
     eval_general(*create_test_series(data), lambda s: "string" + s)
     eval_general(*create_test_series(data), lambda s: s + "string")
 
 
+def test_non_commutative_multiply():
+    # This test checks that mul and rmul do different things when
+    # multiplication is not commutative, e.g. for adding a string to a string.
+    # For context see https://github.com/modin-project/modin/issues/5238
+    modin_series, pandas_series = create_test_series(1, dtype=int)
+    integer = NonCommutativeMultiplyInteger(2)
+    # It's tricky to get the non commutative integer class implementation
+    # right, so before we do the actual test, check that the operation
+    # we care about is really not commmutative in pandas.
+    assert not (integer * pandas_series).equals(pandas_series * integer)
```

Comment:
again, this should be a separate test that checks implementation of non-commutative int

Reply:
Done.